src/test/java/use_case/execute_buy/ExecuteBuyInteractorTest.java

```diff
@@ -59,6 +61,33 @@ void successTest() throws ValidationException {
         }
     }
 
+    // TODO: Fail the test, as invalid input is not handled
```

Comment:
After checking for invalid input as I mentioned above, this test should work as intented. In addition, write a test case where the input quantity is not an integer at all (double, String, DROP TABLE, etc.)

Reply:
I guess we should put relevant tests in the ControllerTest, instead of Interactor.